rs/ingress_manager/benches/build_payload.rs

```diff
@@ -348,6 +349,12 @@ fn validate_payload(criterion: &mut Criterion) {
     group.finish()
 }
 
-criterion_group!(benches, build_payload, validate_payload);
+criterion_group! {
+    name = benches;
+    // Flamegraphs can be generated by passing the `--profile-time SECONDS` argument
+    // to the benchmark. The SVG files can be found in the bazel output directory.
```

Comment:
What happens if we don't pass the `--profile-time` flag? Will a flamegraph not be generated in that case?

Also, what happens if we pass something larger than 10 seconds (which is the duration of each benchmark right now) as `--profile-time`?

Reply:
From criterion's reference on profilers: https://bheisler.github.io/criterion.rs/book/user_guide/profiling.html

If you don't pass `--profile-time`, no flamegraphs are generated, and the benchmarks will run with the default criterion instrumentation. So without the argument, nothing is different from before. 

When you pass a `--profile-time`, it will simply override any benchmark durations you set. So it doesn't matter if you set it to 10, it will run longer if you want.